### ISSUE TITLE
GPU Meter not showing any signs of life after completing tutorial.

### ISSUE THREAD

**Issue by Sprintracer24** [raiguard/ModernGadgets#53]
[DUMPandGlobalSettings.zip](https://github.com/raiguard/ModernGadgets/files/1477018/DUMPandGlobalSettings.zip)
So, all the other gadgets work(CPU Usage, Disks, Network) or the ones I choose to use at least. I have completely went through the tutorial and double checked. I didn't have to change any of the sensor ID's at all. I actually did have this working fine at one point a while back but that was before I had a major issue with Windows 10 a while back and basically wiped my entire computer and started from scratch.. yeah Windows 10 is awful sometimes. Anyways, I feel like fixing it finally and will leave the Dump file and the global settings file like the tutorial says too. If anyone could give me some insight, that would be nice because I really enjoy this gadget as it has become a necessity. Thanks :)

**Comment by raiguard** [comment]
Is it showing the HWiNFO error message, or is it just all zeros? It would be helpful to provide a screenshot of what you're seeing in GPU Meter. Also, open Rainmeter's log (right-click the tray icon and click "about") and tell me what it says there.

**Comment by Sprintracer24** [comment]
[GPU meter screenshots.zip](https://github.com/raiguard/ModernGadgets/files/1490205/GPU.meter.screenshots.zip)

Alright, so I am not getting any HWiNFO errors that are blatant except for the list of errors in my Rainmeter ABOUT screenshot.. which is new to me. I've included what my actual GPU meter looks like and the HWiNFO config tool. I am on version 4.0.0 r2746 of Rainmeter too btw. I hope this can clear things up! Thank you for the reply

**Comment by raiguard** [comment]
First off, make certain that HWiNFO itself is configured correctly. Right-click on HWiNFO’s tray icon and click “settings”. Make sure that your settings match the settings shown in the tutorial. After that, restart HWiNFO, then restart Rainmeter. If that does not help, completely reinstall ModernGadgets and make certain that it installs the HWiNFO plugin as well.

I would also recommend upgrading to Rainmeter 4.1, as the next release of ModernGadgets (hopefully coming in the next few weeks) will require it.

**Comment by Sprintracer24** [comment]
WOO!! We got it working! I backed up my skins then uninstalled Rainmeter completely. Uninstalled HWiNFO aswell but I dont think I needed too. Reinstalled HWiNFO then reinstalled Rainmeter with the upgraded 4.1 version. I then opened HWiNFO and boom! All my gadgets now work and its beautiful! So I think it may have been some setting that wasn't right or something.. Anyways, Thank you for your help boss!
